logger/common.go

```diff
@@ -317,6 +349,11 @@ func (l *Logger) Read(
 					// reset head and bytesInBuffer
 					head = 0
 					bytesInBuffer = 0
+					// increment partial flags
+					partialOrdinal += 1
+					if isFirstPartial {
+						isFirstPartial = false
+					}
```

Comment:
what does this block mean?

Reply:
the current message is still partial, so we increment the ordinal and flip isFirstPartial to false if it was true, since then this would be the 2nd partial message